azurerm/data_source_dev_test_virtual_network.go

```diff
@@ -0,0 +1,182 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/services/devtestlabs/mgmt/2016-05-15/dtl"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmDevTestVirtualNetwork() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmDevTestVnetRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validate.NoEmptyStrings,
+			},
+
+			"lab_name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validate.DevTestLabName(),
+			},
+
+			"resource_group_name": azure.SchemaResourceGroupNameForDataSource(),
+
+			"unique_identifier": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"allowed_subnets": {
+				Type:     schema.TypeList,
+				Computed: true,
+				MaxItems: 1,
```

Comment:
this won't have any effect since this field is Computed - as such could we remove this?

```suggestion
```